nuxeo-dm/nuxeo-platform-webapp/src/main/resources/web/nuxeo.war/widgets/summary/description_widget_template.xhtml

```diff
@@ -21,7 +21,9 @@
   </c:if>
   <c:if test="#{!empty currentDocument.dc.created}">
     <span class="creationDate nowrap">
-      <h:outputText value="#{currentDocument.dc.created}" />
+      <nxl:widgetType name="datetime" mode="view"
+        value="#{currentDocument.dc.created}"
+        pattern="#{nxu:basicDateAndTimeFormatter()}" />
     </span>
```

Comment:
:+1: but i'd rather use the same way than for description above, using dynamic type instead, eg:
"nxwt:datetime" instead of "nxl:widgetType name="datetime"" (this is exactly equivalent)